pkg/pillar/zfs/zfs.go

```diff
@@ -19,36 +19,37 @@ import (
 	"github.com/lf-edge/eve/pkg/pillar/hardware"
 	"github.com/lf-edge/eve/pkg/pillar/types"
 	"github.com/lf-edge/eve/pkg/pillar/utils/disks"
-	"github.com/lf-edge/eve/pkg/pillar/vault"
 	"github.com/prometheus/procfs/blockdevice"
 	log "github.com/sirupsen/logrus"
 )
 
 const volBlockSize = uint64(16 * 1024)
 
 var (
-	zfsPath   = []string{"/hostfs", "zfs"}
-	zpoolPath = []string{"/hostfs", "zpool"}
+	// BinaryZFS is the zfs binary
+	BinaryZFS = "zfs"
```

Comment:
Having this as a definition confused me compared just reading the string "zfs" in the code.
If you really want a const for it, then ZFSBinary is more understandable than BinaryZFS. Latter reads like some binary encoding of zfs or whatever.

Reply:
I think that having constant make sense to keep track of direct calls to binary. I moved constant to types package, seems it is better place for them.

Reply:
OK